src/main/java/com/thealgorithms/maths/GoldbachConjecture.java

```diff
@@ -0,0 +1,40 @@
+package com.thealgorithms.maths;
+
+import static com.thealgorithms.maths.PrimeCheck.isPrime;
+import static java.lang.String.format;
+
+import java.util.Scanner;
+
+/**
+ * This is a representation of the unsolved problem of Goldbach's Projection, according to which every
+ * even natural number greater than 2 can be written as the sum of 2 prime numbers
+ * More info: https://en.wikipedia.org/wiki/Goldbach%27s_conjecture
+ * @author Vasilis Sarantidis (https://github.com/BILLSARAN)
+ */
+
+public final class GoldbachConjecture {
+    private GoldbachConjecture() {
+    }
+
+    public static String getPrimeSum(int number) {
```

Comment:
Please return two integers instead of a formatted string, this is more universal